### PR TITLE
Translations google translate

### PR DESCRIPTION
**Pull request by pmjanus**
Made a script to use the google translate api and get translations for some high volume languages, I speak Spanish and the translation seems pretty good for that language but obviously cannot attest to the others. I think it is a good place to start, as it can help people more knowledgeable to get the translations to a better spot.

### CODE REVIEW

**Comment by 0xngmi** [comment]
not very keen on adding all these machine translations, whenever i see an app that is badly translated it gives off a bad vibe and I think people might prefer using the english version rather than a badly translated version

**Comment by pmjanus** [comment]
> not very keen on adding all these machine translations, whenever i see an app that is badly translated it gives off a bad vibe and I think people might prefer using the english version rather than a badly translated version

I agree I would rather use English but there are people who might not be able to and would prefer a machine translation to none. A couple of points:
1.  I can vouch for the Spanish one, I tested that one first since I know the language, and it is a very good translation, although some terms like wallet might vary depending on your country. I cannot vouch for other languages.
2. Maybe we can add them to a folder so translators can use them as a starting point, fixing the machine translations with the english one available should save people time.
3. Maybe we can make available the machine translations as part of a work in progress, so people who have a problem understanding a part of the site could opt in to see the translation, this could be:
 a. a small header on the drop down, the better translations on top, and the work-in-progress ones clearly marked below them.
 b. or a small very thin banner on top/bottom of the site for the work-in-progress ones stating that they are machine translations with a link to the english site.

File: translations/es.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
+{
+  "Common": {
+    "getStarted": "Empezar",
```
Comment (wei3erHase) [inline translations/es.json:3]: Comenzar

File: translations/es.json [hunk 2]
```diff
@@ -0,0 +1,210 @@
+{
+  "Common": {
+    "getStarted": "Empezar",
+    "connectWallet": "Conectar una billetera",
+    "account": "Cuenta",
+    "connectedWith": "Conectado con",
+    "connectedAs": "Conectado como",
+    "network": "Red",
+    "networkNotSupported": "Red no compatible",
+    "unsupported": "no soportado",
```
Comment (wei3erHase) [inline translations/es.json:10]: No soportado

File: translations/es.json [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    "connectWallet": "Conectar una billetera",
+    "account": "Cuenta",
+    "connectedWith": "Conectado con",
+    "connectedAs": "Conectado como",
+    "network": "Red",
+    "networkNotSupported": "Red no compatible",
+    "unsupported": "no soportado",
+    "disconnect": "Desconectar",
+    "error": "No se pudieron cargar los datos",
+    "token": "Simbólico",
```
Comment (wei3erHase) [inline translations/es.json:13]: Token? Tóken (more correct)? Símbolo? (for Symbol)

File: translations/es.json [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    "amount": "Monto",
+    "deposit": "Depósito",
+    "withdraw": "Retirar",
```
Comment (wei3erHase) [inline translations/es.json:16]: Depositar / Retirar
Depósito / Retiro

File: translations/es.json [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    "pause": "Pausa",
```
Comment (wei3erHase) [inline translations/es.json:17]: Pausar (is it an action?)

File: translations/es.json [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    "or": "o",
+    "name": "Nombre",
+    "address": "Dirección",
+    "history": "Historia",
+    "edit": "Editar",
+    "update": "Actualizar",
+    "month": "Mes",
+    "year": "Año",
+    "close": "Cerrar",
+    "goBack": "Regresa",
```
Comment (wei3erHase) [inline translations/es.json:28]: Regresar

Comment (wei3erHase) [reply translations/es.json:28]: Or "Atras

File: translations/es.json [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+    "menu": "Menú",
+    "streamsAndHistory": "Corrientes e Historia",
```
Comment (wei3erHase) [inline translations/es.json:30]: I'd not translate Stream, or call it "Flujos"

File: translations/es.json [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+    "logoAlt": "Logotipo de {nombre}",
+    "transactionSubmitted": "Transacción enviada",
+    "viewOnExplorer": "Ver en {explorador}",
+    "eventTimestamp": "Marca de tiempo del evento",
```
Comment (wei3erHase) [inline translations/es.json:34]: Fecha / Tiempo / "Timestamp" del evento

File: translations/es.json [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+    "moreInfo": "Más información",
+    "oldStream": "Viejo arroyo",
+    "newStream": "Nueva corriente",
```
Comment (wei3erHase) [inline translations/es.json:37]: Stream or Flujo

File: translations/es.json [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+    "stream": "Arroyo",
```
Comment (wei3erHase) [inline translations/es.json:38]: Same here

File: translations/es.json [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+    "payer": "Pagador",
+    "payee": "Tenedor",
```
Comment (wei3erHase) [inline translations/es.json:40]: Proveedor (provider) and Beneficiario (beneficiary)

File: translations/es.json [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+    "streamHistory": "Historial de transmisión",
```
Comment (wei3erHase) [inline translations/es.json:41]: *de Flujos

Comment (wei3erHase) [reply translations/es.json:41]: Or Transmisiones, but make it global translation Stream = Transmisión

File: translations/es.json [hunk 13]
```diff
@@ -0,0 +1,210 @@
...
+    "eventType": "Tipo de evento",
+    "timestamp": "marca de tiempo",
```
Comment (wei3erHase) [inline translations/es.json:43]: Fecha / Timestamp

File: translations/es.json [hunk 14]
```diff
@@ -0,0 +1,210 @@
...
+    "you": "Tú"
+  },
+  "Header": {
+    "connectWallet": "Conectar billetera",
+    "language": "Idioma",
+    "docs": "Documentos"
+  },
+  "Banner": {
+    "heading": "¡Automatice los pagos de salarios recurrentes en criptografía!",
+    "description": "Automatice los txs de salarios, transmitiéndolos por segundo para que los empleados puedan retirarlos cuando lo deseen y no tenga que lidiar con el envío de txs manualmente.",
```
Comment (wei3erHase) [inline translations/es.json:53]: *las txs de salarios

File: translations/es.json [hunk 15]
```diff
@@ -0,0 +1,210 @@
...
+    "close": "Cerrar pancarta"
```
Comment (wei3erHase) [inline translations/es.json:54]: Cerrar

File: translations/es.json [hunk 16]
```diff
@@ -0,0 +1,210 @@
...
+  },
+  "Balances": {
+    "heading": "Saldos",
+    "balance": "Equilibrio",
```
Comment (wei3erHase) [inline translations/es.json:58]: Balance (equilibrio is for standing up)

File: translations/es.json [hunk 17]
```diff
@@ -0,0 +1,210 @@
...
+    "connectWallet": "Conecte la billetera para ver sus saldos",
+    "deposit": "Depositar nuevo token",
+    "noData": "Crear un saldo primero",
```
Comment (wei3erHase) [inline translations/es.json:61]: ???

File: translations/es.json [hunk 18]
```diff
@@ -0,0 +1,210 @@
...
+    "error": "No se pudieron cargar saldos",
+    "toDepleted": "Agotado",
+    "monthlyCost": "Costo mensual",
+    "topup": "Completar",
```
Comment (wei3erHase) [inline translations/es.json:65]: Rellenar

File: translations/es.json [hunk 19]
```diff
@@ -0,0 +1,210 @@
...
+    "streamsDepleted": "Corrientes agotadas",
```
Comment (wei3erHase) [inline translations/es.json:66]: Flujos / Transmisiones, "corrientes" is just not accurate

File: translations/es.json [hunk 20]
```diff
@@ -0,0 +1,210 @@
...
+    "noStreams": "Sin transmisiones",
+    "days": "dias"
```
Comment (wei3erHase) [inline translations/es.json:68]: días

File: translations/es.json [hunk 21]
```diff
@@ -0,0 +1,210 @@
...
+  },
+  "Streams": {
+    "heading": "Corrientes",
```
Comment (wei3erHase) [inline translations/es.json:71]: 👁️

File: translations/es.json [hunk 22]
```diff
@@ -0,0 +1,210 @@
...
+    "totalStreamed": "Transmitido total",
+    "withdrawable": "extraíble",
+    "send": "Enviar",
+    "modify": "Modificar",
+    "paused": "en pausa",
+    "cancel": "Cancelar",
+    "outOfFunds": "sin fondos",
+    "outgoingStream": "Corriente saliente",
+    "incomingStream": "Corriente entrante",
+    "incomingStreamFrom": "Transmisión entrante de",
```
Comment (wei3erHase) [inline translations/es.json:87]: Standarize!

File: translations/es.json [hunk 23]
```diff
@@ -0,0 +1,210 @@
...
+    "disperse": "Dispersar",
+    "withdrawAnotherWallet": "Retirar otra billetera",
+    "sendAll": "Envia todo",
+    "editName": "Editar el nombre de la dirección del beneficiario"
```
Comment (wei3erHase) [inline translations/es.json:91]: Editar el nombre del beneficiario

File: translations/es.json [hunk 24]
```diff
@@ -0,0 +1,210 @@
...
+  },
+  "Table": {
+    "exportCSV": "Exportar CSV",
+    "rowsPerPage": "Filas por página",
+    "next": "próximo",
+    "previous": "Anterior",
+    "userName": "Nombre",
+    "address": "Dirección",
+    "tokenSymbol": "Simbólico",
+    "amountPerSec": "Monto",
```
Comment (wei3erHase) [inline translations/es.json:122]: Monto por segundo

File: translations/es.json [hunk 25]
```diff
@@ -0,0 +1,210 @@
...
+    "error": "No se pudieron cargar los beneficiarios"
+  },
+  "Table": {
+    "exportCSV": "Exportar CSV",
+    "rowsPerPage": "Filas por página",
+    "next": "próximo",
+    "previous": "Anterior",
+    "userName": "Nombre",
+    "address": "Dirección",
+    "tokenSymbol": "Simbólico",
```
Comment (wei3erHase) [inline translations/es.json:121]: Fix as Token / Símbolo

File: translations/es.json [hunk 26]
```diff
@@ -0,0 +1,210 @@
...
+    "addressName": "Nombre de la dirección",
+    "amount": "Monto",
+    "age": "Edad"
+  },
+  "Forms": {
+    "tokenToDeposit": "¿Qué token quieres depositar?",
+    "availableForDeposit": "Disponible para depósito",
+    "amountToDeposit": "¿Cuánto quieres depositar en total?",
+    "max": "MÁX.",
+    "approveOnWallet": "Aprobar en Wallet",
```
Comment (wei3erHase) [inline translations/es.json:137]: Wallet = Billetera

File: translations/es.json [hunk 27]
```diff
@@ -0,0 +1,210 @@
...
+    "streamDuration": "Duración de la transmisión",
+    "createStream": "Crear transmisión",
+    "delete": "Borrar",
+    "addStream": "Agregar transmisión",
+    "requiredField": "Este campo es obligatorio",
+    "validAddress": "Introduce una dirección válida",
+    "validNumber": "Introduce un número válido",
+    "withdrawOnBehalf": "Retirar en nombre de otra billetera",
+    "tokensError": "No se pudo cargar la lista de tokens",
+    "enterPayerAddress": "Ingrese la dirección del pagador",
```
Comment (wei3erHase) [inline translations/es.json:157]: "pagador" is not of generalized use

File: translations/es.json [hunk 28]
```diff
@@ -0,0 +1,210 @@
...
+  "OnboardWalletConnect": {
+    "header": "Conecta tu billetera",
+    "initializing": "Inicializando",
+    "footer": "Conectar una billetera no mueve fondos",
+    "welcome": "¡Bienvenido!",
+    "description": "Cree flujos de duración indefinida y simplemente extraiga dinero de un grupo, lo que hace posible recargar todos los flujos en una sola operación y proporcionar el dinero que sea necesario para mantenerlos.",
+    "learnMore": "Obtenga más información en nuestros documentos"
+  },
+  "Onboard": {
+    "worksWithAllTokens": "Funciona con todas las fichas.",
```
Comment (wei3erHase) [inline translations/es.json:179]: "fichas" != token

**Review by wei3erHase — COMMENTED** [review]
Good approach!
Many tiny errors on Spanish translation, i've left some comments on it

File: translations/es.json (hunk 23 above)
Comment (pmjanus) [reply translations/es.json:91]: I think this one is fine, can you review? I have been closing without comments the one I agree with. 
Also thank you so much!

File: translations/es.json (hunk 27 above)
Comment (pmjanus) [reply translations/es.json:157]: proveedor?

**Pull request closed without merging**